orchestrator/careplancontributor/service.go

```diff
@@ -20,16 +21,33 @@ import (
 const LandingURL = "/contrib/"
 const CarePlanServiceOAuth2Scope = "careplanservice"
 
-func New(config Config, sessionManager *user.SessionManager, carePlanServiceHttpClient *http.Client, didResolver addressing.StaticDIDResolver) *Service {
+func New(config Config, sessionManager *user.SessionManager, carePlanServiceHttpClient *http.Client, didResolver addressing.StaticDIDResolver) (*Service, error) {
+	fhirURL, _ := url.Parse(config.FHIR.BaseURL)
 	cpsURL, _ := url.Parse(config.CarePlanService.URL)
 	carePlanServiceClient := fhirclient.New(cpsURL, carePlanServiceHttpClient, coolfhir.Config())
+	var transport http.RoundTripper
```

Comment:
I think we have the same code for the CarePlanService authenticating to its FHIR API. Do we need a shared function there, that works on a common config struct? They are probably the same (it's CPS connecting to local FHIR API, and CPC connecting to local FHIR API, so I'd say the same thing, although the might be separate FHIR APIs).